tests/test_pipenv.py

```diff
@@ -440,6 +440,25 @@ def test_windows_pinned_pipfile(self):
             assert 'tablib' in p.lockfile['default']
 
 
+    @pytest.mark.e
+    @pytest.mark.install
+    @pytest.mark.vcs
```

Comment:
Would you be so kind as to add the `resolver` mark as well, so we can track this down when we do make substantial changes to the resolver?

Reply:
Will do.